syntax = "proto3";

service ChatService {
    rpc CreateAccount(Request) returns (Response) {}
    rpc LogIn(Request) returns (Response) {}
    rpc SendMessage(Request) returns (Response) {}
    rpc ShowAccounts(Request) returns (Response) {}
    rpc DeleteAccount(Request) returns (Response) {}
    rpc ReceiveMessage(Request) returns (Response) {}
    rpc LogOut(Request) returns (Response) {}
}

message Request {
    string request = 1;
}

message Response {
    string response = 1;
}